rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1491,12 +1555,18 @@ public ResponseEntity<EntityModel<Attachment>> patchProjectAttachmentInfo(
             @PathVariable("id") String id,
             @Parameter(description = "Attachment ID.")
             @PathVariable("attachmentId") String attachmentId,
-            @RequestBody Attachment attachmentData
+            @RequestBody Attachment attachmentData,
+            @Parameter(description = "Comment message.")
+            @RequestParam(value = "comment", required = false) String comment
     ) throws TException {
         final User sw360User = restControllerHelper.getSw360UserFromAuthentication();
         final Project sw360Project = projectService.getProjectForUserById(id, sw360User);
         Set<Attachment> attachments = sw360Project.getAttachments();
+        sw360User.setCommentMadeDuringModerationRequest(comment);
         Attachment updatedAttachment = attachmentService.updateAttachment(attachments, attachmentData, attachmentId, sw360User);
+        if (!restControllerHelper.isWriteActionAllowed(sw360Project, sw360User) && comment == null) {
+            return new ResponseEntity(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?